service_invocation/csharp/http/checkout/Program.cs

```diff
@@ -9,7 +9,7 @@
 // Adding app id as part of the header
 client.DefaultRequestHeaders.Add("dapr-app-id", "order-processor");
 
-for (int i = 1; i <= 10; i++) {
+for (int i = 1; i <= 100; i++) {

```

Comment:
I worry a tiny bit that mainstream invoke quickstart users think this goes on forever when we loop 100 times.  And will this cause instability to these tests?

Reply:
@paulyuk fair concern. We need to allow enough time to see the expected behavior and introduce a fault. Perhaps somewhere along the lines of 20 is better?